myModule.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
yes, the entire tmpdir should be deleted (`recursive = TRUE`) on exit if one is being created